charts/handbrake/1.0.0/questions.yaml

```diff
@@ -0,0 +1,703 @@
+                        default: "letsencrypt-prod"
+                        enum:
+                          - value: ""
+                            description: "No Encryption/TLS/Certificates"
+                          - value: "letsencrypt-prod"
+                            description: "Letsencrypt Production Certificates"
+                          - value: "letsencrypt-staging"
+                            description: "Letsencrypt Test Certificates"
+                          - value: "wildcard"
+                            description: "Global Wildcard Certificate"
+                          - value: "selfsigned"
+                            description: "Self-Signed Certificate"
+                          - value: "existingcert"
+                            description: "Existing Certificate"
+                        show_subquestions_if: "existingcert"
+                        subquestions:
+                          - variable: existingcert
+                            label: "Existing Certificate Name"
+                            schema:
+                              type: string
+                              default: ""
+                    - variable: authForwardURL
+                      label: "Forward Authentication URL"
+                      schema:
+                        type: string
+                        default: ""
+        - variable: vnc
+          label: "VNC Ingress Configuration"
+          group: "Ingress"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable VNC Ingress"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: type
+                      label: "Select Ingress Type"
+                      schema:
+                        type: string
+                        default: "TCP"
+                        required: true
+                        editable: false
+                        enum:
+                          - value: "TCP"
+                            description: "Plain TCP Connections"
+                    - variable: entrypoint
+                      label: "Select Entrypoint"
+                      schema:
+                        type: string
+                        default: "vnc"
+                        required: true
+                        enum:
+                          - value: "vnc"
+                            description: "VNC: port 5900"
+                    - variable: certType
+                      label: "Select Certificate Type"
+                      schema:
+                        editable: false
+                        type: string
+                        default: ""
+                        enum:
+                          - value: ""
+                            description: "No Encryption/TLS/Certificates"                        
```

Comment:
Don't mess with ingresses you are told are not going to work.
entrypoint VNC doesn't exist and i'm not going to add an ingress for this feature.

Reply:
Note: People can still use nodeport for this though.